scripts/Crawlers/FrdrCrawler.py

```diff
@@ -0,0 +1,532 @@
+                        _get_contents(content["contents"])
+                    else:
+                        continue
+                else:
+                    continue
+
+        # instantiate the file_size.json transfer
+        file_sizes_path = self.transfer_data(ep_id, ep_path, file_name='file_sizes.json')
+        # open file and read
+        with open(file_sizes_path) as json_file:
+            json_contents = json.load(json_file)
+        # access contents to get list of file types
+        _get_contents(json_contents["contents"])
+        dataset_size = json_contents["size"]
+        globus_path = json_contents["path"]
+        # remove file_sizes
+        os.remove(file_sizes_path)
+        return files_types, dataset_size, globus_path
+
+    def get_all_dataset_description(self):
+        """
+        Builds DATS for all dataset in the field of neuroscience
+        """
+        frdr_dois = []
+        datasets = self._query_frdr()
+        for dataset in datasets:
+            metadata = dataset["metadata"]["frdr:frdr"]
+
+            # TODO: to be updated: we are now testing 1 dataset
+            if "An End-to-end System for Automatic Characterization of Iba1 Immunopositive Microglia in Whole Slide Imaging: Data" in metadata["dc:title"]:
+
+                files_types, ds_size, globus_path = \
+                    self.get_all_files_description(metadata["frdr:globusEndpointName"],
+                                                   metadata["frdr:globusEndpointPath"])
+
+                frdr_dois.append(
+                {
+                    "identifier": {
+                        "identifier": metadata["dc:identifier"][0],
+                        "identifierSource": "DOI",
+                    },
+                    "concept_doi": metadata["dc:identifier"][1],
+                    "latest_version": metadata["dc:date"][0],
+                    "title": metadata["dc:title"],
+                    "creators": list(
+                        map(lambda x: {"name": x}, metadata["dc:creator"])
+                    ),
+                    "description": metadata["dc:description"],
+                    "version": metadata["dc:version"]
+                    if "dc:version" in metadata.keys()
+                    else "None",
+                    "licenses": [
+                        {
+                            "name": metadata["dc:rights"]
+                        }
+                    ],
+                    "keywords": list(map(lambda x: {"value": x}, metadata["dc:subject"])),
+                    "distributions": [
+                        {
+                            "formats": [
+                                file_format.upper()
+                                for file_format in files_types
+                                # Do not modify specific file formats.
+                                if files_types not in ["NIfTI", "BigWig"]
+                            ],
+                            "size": str(ds_size),
```

Comment:
`ERROR    scripts.dats_validator.validator:validator.py:57 2 Validation error in distributions.0.size: '14892864859' is not of type 'number'
`
Change to `"size": int(ds_size),`